container.if

```diff
@@ -864,3 +864,2218 @@ interface(`container_spc_rw_pipes',`
 
 	allow $1 spc_t:fifo_file rw_inherited_fifo_file_perms;
 ')
+
+ifndef(`distro_ubuntu',`
```

Comment:
Same here, all these interfaces are defined in fedora-selinux/selinux-policy repository. Please use "ifdef"

Reply:
There doesn't seem to be any predefined macros that for example separate ubuntu from debian from RedHat/Fedora.

For example __unix__ is defined in gcc...but I see no distro definitions.

ifdef(`__unix__',`
Works...but I'm guessing this would also work in RH/Fedora as well.

Any suggestions on how we can define different distros?

Reply:
Done